From b254e432f53cdbb4a16c03718fdf67c702c6e733 Mon Sep 17 00:00:00 2001
From: Liu Haitao <haitao.liu@windriver.com>
Date: Thu, 15 Sep 2022 06:53:58 +0000
Subject: [PATCH] rpm: empty RPMTEST_SETUP define

+cp: cannot stat '': No such file or directory
+chmod: cannot access 'testing/': No such file or directory
+error: failed to load macro file mtest

Signed-off-by: Liu Haitao <haitao.liu@windriver.com>
---
 tests/local.at | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/tests/local.at b/tests/local.at
index 2e9bfa37c..ef0dd493e 100644
--- a/tests/local.at
+++ b/tests/local.at
@@ -1,15 +1,6 @@
 AT_TESTED([rpm rpmbuild rpmquery])
 
 m4_define([RPMTEST_SETUP],[[
-if ! [ -d testing/ ]; then
-    cp -aP "${RPMTEST}" .
-    chmod -R u+w testing/
-    mkdir -p testing/build
-    ln -s ../data/SOURCES testing/build/
-fi
-export RPMTEST="${PWD}/testing"
-export TOPDIR="${RPMTEST}/build"
-export HOME="${RPMTEST}"
 ]])
 
 m4_define([RPMDB_INIT],[
-- 
2.31.1

